platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/MapRenderer.java

```diff
@@ -121,12 +122,38 @@ private native void nativeInitialize(MapRenderer self,
   private void updateFps() {
     frames++;
     long currentTime = System.nanoTime();
-    double fps = 0;
-    if (currentTime - timeElapsed >= 1) {
-      fps = frames / ((currentTime - timeElapsed) / 1E9);
+    long timeCost = currentTime - timeElapsed;
+
+    if (timeCost >= 1) {
```

Comment:
In addition to the comment above, I believe that this check and `frames` field are obsolete in general - I can't think of a scenario where we'd render more than one frame in a nanosecond. One can only dream...